source/api/presentation/3.0/index.md

```diff
@@ -1074,11 +1074,11 @@ The Canvas represents an individual page or view and acts as a central point for
 
 Every Canvas _SHOULD_ have a `label` to display. If one is not provided, the client _SHOULD_ automatically generate one for use based on the Canvas's position within the current Sequence.
 
-A Canvas _MUST_ have a rectangular aspect ratio (described with the `height` and `width` properties) and/or a `duration` to provide an extent in time. These dimensions allow resources to be associated with specific regions of the Canvas, within the space and/or time extents provided. Content _MUST NOT_ be associated with space or time outside of the Canvas's dimensions, such as at coordinates below 0,0, greater than the height or width, before 0 seconds, or after the duration.
+Content resources are associated with the Canvas via Web Annotations.  Content that is to be rendered as part of the Canvas _MUST_ be associated by an Annotation with the "painting" `motivation`. These Annotations are recorded in the `items` of one or more AnnotationPages, refered to in the `items` array of the Canvas. If the Annotation should be rendered quickly, in the view of the publisher, then it _SHOULD_ be embedded within the Manifest directly.  Other AnnotationPages can be referenced with just their `id`, `type` and optionally a `label`, and clients _SHOULD_ dereference these pages to discover further content.  Content in this case includes media assets such as images, video and audio, textual transcriptions or editions of the Canvas, as well as commentary about the object represented by the Canvas.  These different uses _MAY_ be split up across different AnnotationPages.
 
-Renderers _MUST_ scale content into the space represented by the Canvas, and _SHOULD_ follow any `timeMode` adjustment provided for time-based media.  If the Canvas represents a view of a physical object, the spatial dimensions of the Canvas _SHOULD_ be the same scale as that physical object, and images _SHOULD_ depict only the object.
+A Canvas _MUST_ have a rectangular aspect ratio (described with the `height` and `width` properties) and/or a `duration` to provide an extent in time. These dimensions allow resources to be associated with specific regions of the Canvas, within the space and/or time extents provided. Content _MUST NOT_ be associated with space or time outside of the Canvas's dimensions, such as at coordinates below 0,0, greater than the height or width, before 0 seconds, or after the duration. Content resources that have dimensions which are not specified in the Canvas _MUST NOT_ be associated with the Canvas. For example it is an error to associate a Video resource (with height, width and duration) with a Canvas that does not have all all three dimensions. The resource can be referenced with the `rendering` property, or by Annotations with a `motivation` other than "painting", in the `annotations` property.
```

Comment:
I think this would benefit from a second example:

For example, it is an error to use a `painting` annotation to associate a Video resource (which has height, width and duration) with a Canvas that does not have all all three dimensions, but it is not an error to use a `painting` annotation to associate an image (which has width and height only) with a Canvas that has all of width, height and duration, either for the entire duration of the Canvas or for part of that duration. 

Where the resource has dimensions that are not available in the target Canvas, the resource can be referenced with the `rendering` property, or by Annotations with a `motivation` other than "painting", in the `annotations` property.

(the wording can be tightened up further)